### ISSUE TITLE
Blog posts not fetching

### ISSUE THREAD

**Issue by HammyHavoc** [rahuldkjain/github-profile-readme-generator#504]
**Describe the bug**
I followed the guide, yet the blog posts don't appear on my profile. https://github.com/HammyHavoc/hammyhavoc

According to the repository, the workflow hasn't run. CCing @gautamkrishnar.

**To Reproduce**

1. Generated profile info via tool
2. Followed the guide

**Expected behavior**
Blog posts to appear.

**Screenshots**

**Desktop (please complete the following information):**

- OS: Manjaro
- Browser Firefox
- Version 94.0.1

**Additional context**
I haven't used GitHub Actions before, so there's a chance I may be doing something silly. 5am brain is definitely here.

Join the **Discord Server** for further discussions.

<a href="https://discord.gg/HHMs7Eg">
<img src="https://discord.com/assets/e4923594e694a21542a489471ecffa50.svg" alt="GPRG Discord Server Link" width="300px"/>
</a>

Server Link: https://discord.gg/HHMs7Eg

**Comment by gautamkrishnar** [comment]
hope this is fixed @HammyHavoc ?

**Comment by HammyHavoc** [comment]
Yes, thank you! It's awesome, love it. :- )

In terms of what went wrong, it may have been that I was tired and made a mistake. However, instead of pushing the file generated by the web-based tool, I simply followed the guide and copy-pasted. Worked first time.

In terms of differences between the two methods, the generated YML features quotation marks (`"`) and the guide features apostrophes (`'`). Though not sure that would have been the issue. Adding a dispatch for manual runs to the automatically generated YML file might be good for troubleshooting, it was through that I was able to figure out that there was a syntax problem.
